Makefile

```diff
@@ -46,17 +46,17 @@ server: compose-build
 server-shell: compose-build
 	@docker-compose run $(IMAGE_NAME) /bin/bash
 
-integration-shell: compose-build
```

Comment:
Marking `compose-build` as a pre-requisite for the scripts to run was causing `docker compose build` to be run twice, taking up time when running any of these commands both locally and in CI.